.ci/deps.python-packages.ps1

```diff
@@ -91,7 +91,7 @@ function Install-coala {
 
                     Checkpoint-Pip-Constraints
 
-                    Install-Pip-Requirement 'git+https://github.com/coala/coala#egg=coala'
+                    Install-Pip-Requirement 'git+https://github.com/akshatkarani/coala@nextgen#egg=coala'
```

Comment:
I suppose Temp changes can be removed now since https://github.com/coala/coala/pull/6029 is merged